gatorgrouper/utils/display.py

```diff
@@ -8,15 +8,15 @@ def display_student_groups(student_groups):
     group_counter = 1
     place_counter = 0
     for student_group in student_groups:
-        #print("Group", group_counter)
-        print('\033[0;32m' + '\033[1m' + '\033[4m' + "Group", group_counter)
+        # print("Group", group_counter)
+        print("\033[0;32m" + "\033[1m" + "\033[4m" + "Group", group_counter)
         for student in student_group:
             place_counter += 1
-            #print(student[0])
-            print('\033[0m' + student[0])   # converts the students back to original font
+            # print(student[0])
+            print("\033[0m" + student[0])  # converts the students back to original font
         print()
         group_counter = group_counter + 1
-    logging.info("Found " + str(place_counter) + " students")
+    logging.info("Found %d students", place_counter)
```

Comment:
@Michionlion @aubreypc . I have changed logging to pass `pylint` errors but I would like someone to look over if I have correctly used the correct data type e.g `%d`. Additionally as I discussed with @gkapfham, logging may be unnecessary if we have good test cases and may be deleted. However, this topic does not take priority and can be done later if need be.